page_objects/burger_menu.py

```diff
@@ -1,42 +1,35 @@
+import time
+
 from selenium.webdriver.common.by import By
 from Automation_tests.utilities.web_ui.base_page import BasePage
+from CONSTANTS import about_url
 
 
 class BurgerMenu(BasePage):
     def __init__(self, driver):
         super().__init__(driver)
 
     __burger_menu = (By.XPATH, "//*[@id='react-burger-menu-btn']")
-    __all_items = (By.XPATH, "//*[@id='inventory_sidebar_link']")
     __about = (By.XPATH, "//*[@id='about_sidebar_link']")
     __logout = (By.XPATH, "//*[@id='logout_sidebar_link']")
-    __reset_app_state = (By.XPATH, "//*[@id='reset_sidebar_link']")
-    __about_url = "https://saucelabs.com/"
-    __text_all_items = "ALL ITEMS"
-
-    def get_burger_menu(self):
-        return self._click(self.__burger_menu)
-
-    def get_all_items(self):
-        return self.__all_items
-
-    def get_about(self):
-        return self.__about
-
-    def get_logout(self):
-        return self.__logout
+    __menu_button_container = (By.XPATH, '//*[@id="menu_button_container"]/div/div/div/nav/a')
 
-    def get_reset_app_state(self):
-        return self.__reset_app_state
+    def open_burger_menu(self):
+        return self._click(self._wait_until_element_located(self.__burger_menu)), time.sleep(0.5)
 
-    def get_text(self, burger_menu_item):
-        return self.read_text(burger_menu_item)
+    def list_of_burger_menu(self):
+        list_of_burger_menu = []
+        elements = self._wait_until_elements_located(self.__menu_button_container)
+        for element in elements:
+            list_of_burger_menu.append(element.text)
+        return list_of_burger_menu
```

Comment:
good. But you can move this function to BasePage. 
In BasePage implement get_texts. ANd than use it here

Reply:
Also, you can shorter it. Just use list comprehension